doc/src/manual/arrays.md

```diff
@@ -801,7 +801,7 @@ elements are stored contiguously in column-major order (see additional notes in
 [Performance Tips](@ref man-performance-tips)). The [`Array`](@ref) type is a specific instance
 of `DenseArray`;  [`Vector`](@ref) and [`Matrix`](@ref) are aliases for the 1-d and 2-d cases.
 Very few operations are implemented specifically for `Array` beyond those that are required
-for all `AbstractArrays`s; much of the array library is implemented in a generic
+for all `AbstractArrays`; much of the array library is implemented in a generic
```

Comment:
I think it should be `` `AbstractArray`s ``

Reply:
Yes sir you are right.
I am from India.Can you tell me how I can participate in GSOC.How can I become the part of Julia organization.

Reply:
Please ask questions on https://discourse.julialang.org/.